seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -183,7 +183,7 @@ def test_saved_variant_data(self, mock_analyst_group):
         self.assertSetEqual(set(response_json.keys()), response_keys)
         self.assertEqual(len(response_json['savedVariantsByGuid']), 2)
         project = response_json['projectsByGuid'][PROJECT_GUID]
-        self.assertSetEqual(set(project.keys()), {'variantTagTypes', 'variantFunctionalTagTypes'})
+        self.assertSetEqual(set(project.keys()), {'variantTagTypes', 'variantFunctionalTagTypes', 'projectGuid'})
```

Comment:
this is technically a change to functionality but it makes the shared functionality substantially easier, and theres no harm to returning the project guid